draft-ietf-rtcweb-jsep.xml

```diff
@@ -1044,6 +1083,13 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             be rolled back.
             </t>
 
+            <t>A rollback will disassociate any RtpTransceivers from m=
+            sections that were associated during the application of the session
```

Comment:
maybe add an xref to where the association happens (since it occurs after this text)